facade/servicetemplate.go

```diff
@@ -357,19 +335,19 @@ func (f *Facade) DeployService(ctx datastore.Context, poolID, parentID string, o
 		}
 	}
 
-	return f.deployService(ctx, tenantID, svc.ID, svc.DeploymentID, poolID, overwrite, svcDef)
+	var statusUpdater = func(status string) {}
```

Comment:
Yes.  The facade.DeployService routine is about adding a single service to an existing service tree. We don't monitor the status of those operations, so there's no need to update the status.

Actually, I don't _think_ we use DeployService any more.  We could possibly remove it.  It was part of the old zenpack installation process, but now we use MigrateService via the service migration library.